components/tabs/tab.js

```diff
@@ -21,17 +23,36 @@ class Tab extends TabMixin(LitElement) {
 
 	static get styles() {
 		const styles = [ css`
-			.d2l-tab-text {
-				overflow: hidden;
+			.d2l-tab-text-inner-content {
+				display: flex;
 				padding: 0.1rem;
-				text-overflow: ellipsis;
-				white-space: nowrap;
 			}
-			:host(:${unsafeCSS(getFocusPseudoClass())}) .d2l-tab-text {
+			:host(:${unsafeCSS(getFocusPseudoClass())}) .d2l-tab-text-inner-content {
 				border-radius: 0.3rem;
 				color: var(--d2l-color-celestine);
 				outline: 2px solid var(--d2l-color-celestine);
 			}
+			:host(:${unsafeCSS(getFocusPseudoClass())}) ::slotted(d2l-icon) {
```

Comment:
Might be good to have a vdiff case for the `d2l-icon` unfocused/focused states given that we are applying special styles.

Reply:
Ignore... I see you have it, just hasn't been merged into this branch yet.

Reply:
Yup! I added a few for the icon:
- [one for focus](https://github.com/BrightspaceUI/core/pull/5546/files#diff-75c022233596f82322b044451ff6afb071784d5b22acd1e44ceac4d8358cdd4f)
- [hover when not selected](https://github.com/BrightspaceUI/core/pull/5546/files#diff-341c679e62db38e865443a3d3b673ffe430fd097ebb4076b84caa016896d6ccc)
- [hover when selected](https://github.com/BrightspaceUI/core/pull/5546/files#diff-4e314f877305fc96a8cbaab0ad96e7ae9292ed9ad40d9d4cdba0272c0de44d0f)

Let me know if there's a particular one I should add. There isn't one that has an icon that isn't selected/hover/focused so I can sub an icon in in one of the other slots to capture that.